ci-operator/config/openshift/assisted-installer-agent/openshift-assisted-installer-agent-master__edge.yaml

```diff
@@ -101,18 +111,21 @@ tests:
     - ref: assisted-baremetal-images-publish
 - as: lint
   commands: |
```

Comment:
probably need to apply one more fix here, waiting for the last rehearsal to finish

Reply:
```
 level=error msg="Running error: context loading failed: no go files to analyze" 
```
not sure why it fails with this error, maybe your fix will work :)

Reply:
yep, looks like it worked :)